contracts/ExoticPositionalMarkets/GamesQueue.sol

```diff
@@ -0,0 +1,120 @@
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/security/PausableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+// internal
+import "../utils/proxy/solidity-0.8.0/ProxyOwned.sol";
+import "../utils/proxy/solidity-0.8.0/ProxyPausable.sol";
+
+/* 
+    Storage for games (created or resolved)
+*/
+contract GamesQueue is Initializable, ProxyOwned, ProxyPausable {
```

Comment:
No reviewing as its not being released yet, but curious to understand why this is needed.

Reply:
I wanted to separate from the consumer  contract because of two things:

- there will be more development on odds etc. so wanted to separate storage (thinking in advance)
- there is a really great chance that we will go with `contract code too big` if I use this all in one contract (Consumer.sol) so I separate it early on, because of the first point.